app/views/dashboard.pug

```diff
@@ -38,7 +39,9 @@ html(lang="en")
                         div(style='position:absolute;width:100%;height:20%;left:0%;top:80%;')
                             button.btn.btn-default(type='submit' style='width:100%;height:100%;left:0%;top:0%;') Logout
 
-                div(id='mainDiv' style='position:absolute;width:90%;height100%;left:10%;top:0%;border-style:solid;')
+                div(id='mainDiv' style='position:absolute;width:90%;height100%;left:10%;top:0%;border-style:solid; z-index:1')

```

Comment:
mainDiv is suppose to be the div we reuse for every tab that doesn't use a pop-up. I think the idea was that the pop-up would overlay the mainDiv at all times but with the glitches that didn't work out. That is how I fixed the glitch, is I hide the main div whenever a pop-up is up.